src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,166 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int EMPTY_SIZE = 0;
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    public MyLinkedList() {
```

Comment:
The constructor is initializing 'first' and 'last' to null and 'size' to 0, which is redundant since these are the default values for object references and integers in Java.